examples/index.html

```diff
@@ -4,21 +4,23 @@
   <title>Thebe Lab examples</title>
   <link rel="stylesheet" type="text/css" href="index.css" />
 
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js">
+  </script>
+
   <!-- Configure and load Thebe !-->
   <script type="text/x-thebe-config">
     {
       bootstrap: true,
       binderOptions: {
-        repo: "binder-examples/requirements",
```

Comment:
TODO: undo this change. ipyleaflet is needed for that example. It should probably be added to `binder-examples/requirements`, or we could make a new `binder-examples/thebelab-demo`.